### PR TITLE
Raj210 kumari patch 2

### PR DESCRIPTION
**Pull request by Raj210Kumari**

### CODE REVIEW

File: data/Raj210Kumari.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  "links": [
+    {
+      "name": "Follow me on Github",
+      "url": "https://github.com/Raj210Kumari",
+      "icon": "FaGithub"
+    },
+    {
+      "name": "Checkout my portfolio",
+      "url": "http://raj210kumari.github.io/",
+      "icon": "FaWeb"
```
Comment (senali-d) [inline data/Raj210Kumari.json:14]: `FaWeb` is not a valid icon. You can find any relevant icon from here.
https://linkfree.eddiehub.io/icons

**Review by senali-d — CHANGES_REQUESTED** [review]

**Review by devvsakib — CHANGES_REQUESTED** [review]
line14, icon name not valid. please update your file

File: data/Raj210Kumari.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
+{
+  "name": "Rajnandani Kumari",
+  "type": "personal",
+  "bio": "Lawyer turned **Digital Nomad**. Travel enthusiast Yoga & Peloton aficionado",
```
Comment (kumarsonsoff3) [inline data/Raj210Kumari.json:4]: Please update it with your info/bio. 🛠

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]

File: data/Raj210Kumari.json [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  "links": [
+    {
+      "name": "Follow me on Github",
+      "url": "https://github.com/Raj210Kumari",
+      "icon": "FaGithub"
+    },
+    {
+      "name": "Checkout my portfolio",
+      "url": "http://raj210kumari.github.io/",
+      "icon": "FaWeb"
```
Comment (Panquesito7) [inline data/Raj210Kumari.json:14]: You could use `FaGlobe` for this, as `FaWeb` is not a valid icon as @senali-d said.
```suggestion
      "icon": "FaGlobe"
```

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Pull request merged**